scripts/vet.sh

```diff
@@ -86,18 +86,21 @@ for MOD_FILE in $(find . -name 'go.mod'); do
   popd
 done
 
-# - Collection of static analysis checks
-SC_OUT="$(mktemp)"
-staticcheck -go 1.19 -checks 'all' ./... > "${SC_OUT}" || true
+for MOD_FILE in $(find . -name 'go.mod'); do
+  MOD_DIR=$(dirname ${MOD_FILE})
+  pushd ${MOD_DIR}
+  # - Collection of static analysis checks
+  SC_OUT="$(mktemp)"
+  staticcheck -go 1.19 -checks 'all' ./... >"${SC_OUT}" || true
```

Comment:
Maybe just move this into the previous for loop instead of adding another one?

Reply:
Done